<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>RÃ©partition des colis par destination</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>France</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="twFrance">
         <column>
          <property name="text">
           <string>Destinataire</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Poids</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Volume</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Espagne</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="twEspagne">
         <column>
          <property name="text">
           <string>Destinataire</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Poids</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Volume</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Allemagne</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="twAllemagne">
         <column>
          <property name="text">
           <string>Destinataire</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Poids</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Volume</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
